Update lessons.json

JulianneMorgan opened the pull request:
Added another Lesson for Getting Started. Added more content to the What Is Isidore lesson.

JulianneMorgan committed:
Update lessons.json

Added another Lesson for Getting Started. Added more content to the What Is Isidore lesson.

davidpbauer commented on lessons.json:
```diff
@@ -1,24 +1,193 @@
+            "section1title" : "Step 1: What is Isidore?",
             "section1content" : "Watch this 2 minute video for an introduction to Isidore.",
             "section2title" : "Step 2: Test Your Knowledge",
             "section2content" : "Now it's time to prove you watched the video. Complete the Quick Quiz below and earn a badge."
```
You need a comma on this line now that it isn't the last item in the list.

davidpbauer commented on lessons.json:
```diff
@@ -1,24 +1,193 @@
+            "section1title" : "Step 1: What is Isidore?",
+            "section1content" : "Watch this 1 minute video to learn how to log in and add a profile picture.",
+            "section2title" : "Step 2: Test Your Knowledge",
+            "section2content" : "Now it's time to prove you watched the video. Complete the Quick Quiz below to earn a badge. Earn an elite badge by going above and beyond by recording your name pronunciation."
```
You need a comma at the end of this line.

davidpbauer commented on lessons.json:
```diff
@@ -1,24 +1,193 @@
+            "aside3title" : "Did you know?",
+            "aside3content" : "Our Isidore homagepage has looked mighty different over the years. See all of Isidore's looks: https://drive.google.com/file/d/1YkDqB6d2oNHEmxASZ1dErPHdlRmwJU9V/view?usp=sharing"
+        } 
+   ],
```
No comma here and need to end this object with a curly brace before jumping into Isidore 101.

davidpbauer committed:
Syntax fixes

There were a few missing commas and the new lesson needed to be started with a curly brace.

davidpbauer reviewed (commented):
Looks good except for a couple of commas and the Isidore 101 lesson needed to be in its own object.
